src/deviceApiCalls/transports/apple.transport.js

```diff
@@ -28,32 +27,10 @@ export class AppleTransport extends DeviceApiTransport {
                 if (this.config.isDDGTestMode) {
                     console.log('MissingWebkitHandler error for:', deviceApiCall.method)
                 }
-                if (deviceApiCall instanceof GetRuntimeConfigurationCall) {
-                    return deviceApiCall.result(appleSpecificRuntimeConfiguration(this.config))
-                }
```

Comment:
This is the core of the change. We just drop this conditional and the rest just flows as usual.